employees/views.py

```diff
@@ -317,8 +317,11 @@ def render_to_response(self, context: dict, **response_kwargs: Any) -> HttpRespo
 
 
 @method_decorator(login_required, name="dispatch")
-@method_decorator(check_permissions(allowed_user_types=[CustomUser.UserType.ADMIN.name]), name="dispatch")
-class ExportReportsInProjectView(DetailView):
+@method_decorator(
+    check_permissions(allowed_user_types=[CustomUser.UserType.ADMIN.name, CustomUser.UserType.MANAGER.name]),
+    name="dispatch",
+)
+class ExportReportsInProjectView(DetailView, UserIsManagerOfCurrentReportProjectMixin):
```

Comment:
Just reverse classes here.

Reply:
@rwrzesien Exactly what I wanted you to see! When they are reversed they don't work :D And I am wondering why :P

Reply:
I have done this by myself, but please remember to update `AccessPermissionsTestCase` if you modify anything regarding access permission.